composer.json

```diff
@@ -59,7 +59,8 @@
     "symfony/service-contracts": "1.1.8"
   },
   "replace": {
-    "guzzlehttp/guzzle": "*"
+    "guzzlehttp/guzzle": "*",
+    "symfony/console": "*"
```

Comment:
I've never user `replace` but from https://getcomposer.org/doc/04-schema.md#replace it sounds a bit dangerous. Like you're telling composer that *this package* replaces guzzle and symfony/console. And I assume your intention is that the app shall fine and use the ones from server, right? So this *can* work if the composer here assumes the same guzzle version as in Nextcloud, but might fail hard otherwise, right? Like here composer assumes guzzle x is available, but in reality version y is shipped.

Reply:
if the libs here can also work with httplug instead of guzzle directly, you may also try https://packagist.org/packages/christophwurst/nextcloud-http-client

Reply:
Yep. Using replace is tricky. symfony/console is luckily only a false positive. feed-io does not depend on symfony/console (actually it does but only for some cli scripts not used by news). guzzlehttp/guzzle maintains a stable api for a while. It seems that all guzzle versions shipped from Nextcloud 16 to Nextcloud 19 do work with feed-io.

Reply:
> guzzlehttp/guzzle maintains a stable api for a while. It seems that all guzzle versions shipped from Nextcloud 16 to Nextcloud 19 do work with feed-io.

I've seen issues with Guzzle in `twofactor_gateway` when we shipped our own version. It was caused by `_idn_uri_convert` or similar. It was somewhat related to https://github.com/guzzle/guzzle/issues/2600#issuecomment-607136269 but I do not remember the details.